src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -50,10 +62,14 @@ const CrossPodWriteForm = () => {
     clearDescription();
```

Comment:
SInce we're going to replace the useField hook, we might be using setDocDescription or something of the like here.

And if we're replacing the useField Hook, we probably will be adding one more parameter to makeHandleFormSubmission to make things work without crashing.